client/src/components/documents/Preview.tsx

```diff
@@ -0,0 +1,53 @@
+import { Dialog, DialogPanel, Button } from "@tremor/react";
+import { Viewer, Worker, ViewMode } from '@react-pdf-viewer/core';
+import { RiFileCloseFill } from "@remixicon/react";
+import { KxDocument } from "../../model";
+import { useEffect, useState } from "react";
+import API from "../../API";
+import { toast } from "../../utils/toaster";
+import { mongoose } from "@typegoose/typegoose";
+import '@react-pdf-viewer/core/lib/styles/index.css';
+
+export default function PreviewDoc(open: boolean, setOpen: (bool: boolean) => void, doc: KxDocument) {
+
+    const [fileUrl, setFileUrl] = useState<string | undefined>(undefined);
+
+    useEffect(() => {
+        const fetchDocuments = async () => {
+            try {
+                if (doc && doc._id) {
+                    const url = await API.getKxFileByID(new mongoose.Types.ObjectId("6730daada436c299749f589e"), "doc_63_info.pdf");
```

Comment:
can you tell me why this Id is static?

Reply:
@dlste  haven't wrote for now the "add file to db"